app/views/govuk_wrapper.scala.html

```diff
@@ -53,7 +53,7 @@
 @linksElement = {
 <link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/palette.css")'>
 <link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/focus.css")'>
-<link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/vat-v1.0.css")'>
+<link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/vat.css?v=0.1")'>
```

Comment:
Not sure if we needed the ?v=0.1 .

Reply:
This allows to refresh the cache by changing a value instead of renaming the file.